### PR TITLE
Update games return value

### PR DESCRIPTION
**Pull request by bkach**
All functions that affect a game should return that game, except for deleting a game (as it no longer exists)

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -6,5 +6,6 @@ require (
 	github.com/gofrs/uuid v3.3.0+incompatible
 	github.com/golang/gddo v0.0.0-20200324184333-3c2cc9a6329d
 	github.com/gorilla/mux v1.7.4
-	github.com/stretchr/testify v1.4.0
+	github.com/stretchr/testify v1.5.1
+	gopkg.in/yaml.v2 v2.3.0 // indirect
```
Comment (bkach) [inline go.mod:10]: These dependencies should've been here to begin with...

File: handlers/createPlayer.go [hunk 2]
```diff
@@ -10,6 +10,7 @@ import (
 
 type CreatePlayerResponse struct {
 	PlayerId string `json:"player_id"`
+	Game truco.Game `json:"game"`
 }
```
Comment (bkach) [inline handlers/createPlayer.go:14]: @ch2ch3 - take a look at these `Response` structs. They'll be what will be marshalled into json and returned. Do they look right to you?

**Comment by bkach** [comment]
[createPlayer should return the game](https://trello.com/c/WGQY3Uu3/12-createplayer-should-return-the-game)

**Comment by bkach** [comment]
[createGame should return the game](https://trello.com/c/UCPwtY5H/11-creategame-should-return-the-game)

**Comment by bkach** [comment]
[dealCards should return the new state of the game](https://trello.com/c/FMiMgqOy/14-dealcards-should-return-the-new-state-of-the-game)

**Comment by bkach** [comment]
[playCard should return the new state of the game](https://trello.com/c/UmHz8fIj/13-playcard-should-return-the-new-state-of-the-game)

**Comment by bkach** [comment]
[Do games need human-edited names? Could the backend generate these?](https://trello.com/c/mjwrgud2/6-do-games-need-human-edited-names-could-the-backend-generate-these)

**Comment by bkach** [comment]
[API suggestions card](https://trello.com/c/xQNua5qX/9-api-suggestions-card)

File: .idea/dictionaries/boriskach.xml [hunk 3]
```diff
@@ -0,0 +1,7 @@
+<component name="ProjectDictionaryState">
+  <dictionary name="boriskach">
+    <words>
+      <w>papi</w>
```
Comment (ch2ch3) [inline .idea/dictionaries/boriskach.xml:4]: adorbs

File: handlers/createGame.go [hunk 4]
```diff
@@ -10,6 +10,7 @@ import (
 
 type CreateGameResponse struct {
 	GameId string `json:"game_id"`
+	Game truco.Game `json:"game"`
```
Comment (ch2ch3) [inline handlers/createGame.go:13]: Could this just be `Game`? I'm assuming `GameId` will be a part of the `Game` object anyway, if that's the case then no point duplicating in the response

File: handlers/createPlayer.go [hunk 5]
```diff
@@ -37,6 +45,7 @@ func CreatePlayerHandler() http.HandlerFunc {
 
 		err = json.NewEncoder(w).Encode(CreatePlayerResponse{
 			PlayerId: playerId,
```
Comment (ch2ch3) [inline handlers/createPlayer.go:47]: What do you think about returning the created `Player` in place of `PlayerId`?

File: handlers/dealCards.go [hunk 6]
```diff
@@ -1,12 +1,17 @@
...
 	"fmt"
 	"github.com/bkach/truco/handlers/util"
 	"github.com/bkach/truco/truco"
 	"net/http"
 )
 
+type DealCardsResponse struct {
+	Game truco.Game `json:"game"`
+}
+
```
Comment (ch2ch3) [inline handlers/dealCards.go:14]: Perf 💯

File: handlers/playCard.go [hunk 7]
```diff
@@ -11,6 +12,10 @@ type PlayCardRequest struct {
 	Card truco.Card `json:"card"`
 }
 
+type PlayCardResponse  struct {
+	Game truco.Game `json:"game"`
+}
+
```
Comment (ch2ch3) [inline handlers/playCard.go:18]: Nailed it 🔨

File: handlers/deletePlayer.go [hunk 8]
```diff
@@ -11,6 +12,10 @@ type DeletePlayerRequest struct {
 	GameId string `json:"game_id"`
 }
 
+type DeletePlayerResponse struct {
+	Game truco.Game `json:"game"`
+}
+
```
Comment (ch2ch3) [inline handlers/deletePlayer.go:18]: Haven't thought through how the frontend will use this one yet, but no harm returning `Game` ✅

**Pull request merged**